src/main/java/cart/admin/application/AdminService.java

```diff
@@ -18,23 +19,29 @@ public AdminService(AdminProductDao adminProductDao) {
         this.adminProductDao = adminProductDao;
     }
 
-    public int createProduct(Product product) {
+    public int createProduct(ProductDto productDto) {
+
+        Product product = new Product(productDto.getName(), productDto.getImageUrl(), productDto.getPrice());
```

Comment:
```suggestion
        Product newProduct = productDto.toEntity() 
```
이런식으로 책임을 전달해도 될 것 같네요 👍 
여기서 조금 더 깊게 생각한다면 toEntity라는 역할은 대부분의 DTO에서도 제공하고싶은 기능일테니 인터페이스로 추상화 하고 제네릭을 통해 무슨 엔티티로 제공할지도 지정해둘 수 있겠죠 👍 

```java
public int createProduct(EntitySupplier<Product> supplier) {
    Product product = supplier.supply()
    ....
}
```
이런식으로 말이죠 👍